src/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -5,61 +5,167 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
 namespace System.IO.IsolatedStorage
 {
+    public partial interface INormalizeForIsolatedStorage
+    {
+        object Normalize();
+    }
+
+    public abstract partial class IsolatedStorage // : System.MarshalByRefObject
+    {
+        protected IsolatedStorage() { }
+        public object ApplicationIdentity { get { throw null; } }
+        public object AssemblyIdentity { get { throw null; } }
+        public virtual long AvailableFreeSpace { get { throw null; } }
+        [System.CLSCompliantAttribute(false)]
+        [System.ObsoleteAttribute("IsolatedStorage.CurrentSize has been deprecated because it is not CLS Compliant.  To get the current size use IsolatedStorage.UsedSize")]
+        public virtual ulong CurrentSize { get { throw null; } }
+        public object DomainIdentity { get { throw null; } }
+        [System.CLSCompliantAttribute(false)]
+        [System.ObsoleteAttribute("IsolatedStorage.MaximumSize has been deprecated because it is not CLS Compliant.  To get the maximum size use IsolatedStorage.Quota")]
+        public virtual ulong MaximumSize { get { throw null; } }
+        public virtual long Quota { get { throw null; } }
+        public System.IO.IsolatedStorage.IsolatedStorageScope Scope { get { throw null; } }
+        protected virtual char SeparatorExternal { get { throw null; } }
+        protected virtual char SeparatorInternal { get { throw null; } }
+        public virtual long UsedSize { get { throw null; } }
+        // protected abstract System.Security.Permissions.IsolatedStoragePermission GetPermission(System.Security.PermissionSet ps);
+        public virtual bool IncreaseQuotaTo(long newQuotaSize) { throw null; }
+        
+        protected void InitStore(System.IO.IsolatedStorage.IsolatedStorageScope scope, System.Type appEvidenceType) { }
+        
+        protected void InitStore(System.IO.IsolatedStorage.IsolatedStorageScope scope, System.Type domainEvidenceType, System.Type assemblyEvidenceType) { }
+        public abstract void Remove();
+    }
+
     public partial class IsolatedStorageException : System.Exception
     {
         public IsolatedStorageException() { }
+        // PermissionSet isn't available in CoreFx
```

Comment:
What happened to the notion that we don't remove members from types, that if a type is in the standard so are all of its members?